src/java/org/apache/cassandra/locator/Endpoints.java

```diff
@@ -88,14 +78,7 @@ public boolean contains(Replica replica)
     // TODO: TR-Review ignoreConflicts is not an acceptable solution here - we need to explicitly resolve them in case of transient/full mismatch
```

Comment:
Is this something we can fix now?

Reply:
Yes, I'll make a note that we resolved non-isolated mismatches from range movements randomly